### ISSUE TITLE
Week view: events on first day of week are not shown

### ISSUE THREAD

**Issue by MrSnoozles** [richardtallent/vue-simple-calendar#80]
Hello,

I'm currently encountering a strange bug. Events on the first day of the week are not shown, if the `showDate` property is set. If it's not set all events are shown correctly. If showDate is set and the date is changed (e.g. go to next week and then back) it works too.

I created a sandbox to test the behaviour:

https://codesandbox.io/s/0p88mx5n1v

With `showDate` property set (event not shown):
![image](https://user-images.githubusercontent.com/17496592/42727364-9462ce28-87a5-11e8-8a51-54165b0adb98.png)

Without `showDate` property (correct behaviour):
![image](https://user-images.githubusercontent.com/17496592/42727371-be4ee8b6-87a5-11e8-9f0e-0be081fe1a15.png)

**Comment by richardtallent** [comment]
Thanks for reporting this! Yes, the `showDate` property can cause problems with event displays if it includes a non-zero component. It should be clearing the time before using it, and it's not doing so in that case.

I'll get this fixed in the next version. In the meantime, you can use something like this:

```JavaScript
(in "data"):
  showDate: today()

(in "methods")
today() {
  const d = new Date()
  d.setHours(0, 0, 0, 0)
  return d
},
```

**Comment by MrSnoozles** [comment]
Thank you for providing the workaround. This works perfectly.

**Comment by richardtallent** [comment]
No problem! I've patched this for release in 3.1, so closing this issue now.

**Comment by PeterPlucinski** [comment]
I'm still having this problem - latest version 3.0.2

**Comment by MrSnoozles** [comment]
This is going to be fixed in the next version (3.1.0 I believe). For now
you have to use the workaround.

Peter Plucinski <notifications@github.com> schrieb am So., 22. Juli 2018,
09:12:

**Comment by PeterPlucinski** [comment]
Ok thanks. I misunderstood Richard's comment. Also thanks for everyones work on this project.
